tests/unit/javascript/conftest.py

```diff
@@ -44,7 +42,7 @@ class JSTester:
     def __init__(self, tab, qtbot, config_stub):
         self.tab = tab
         self.qtbot = qtbot
-        loader = jinja2.FileSystemLoader(os.path.dirname(__file__))
+        loader = jinja2.FileSystemLoader(pathlib.Path(__file__).parent)
```

Comment:
Since we need this multiple times, please do something like `JS_DIR = pathlib.Path(__file__).parent` at module level, and then use `JS_DIR` here and below.